utils/generate-command-code.py

```diff
@@ -399,6 +399,7 @@ def create_command(name, desc):
 with open("commands.c","w") as f:
     f.write("/* Automatically generated by %s, do not edit. */\n\n" % os.path.basename(__file__))
     f.write("#include \"server.h\"\n")
+    f.write("#include \"functions.h\"\n")
```

Comment:
why do we need that?
maybe an indication that something should be moved to server.h?